From 9e6c276905b13fb4a2d795fee5f98d52aecb354b Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Sun, 1 Jun 2014 14:45:49 +0200
Subject: bugfix: Fix buffer overflow in desk_show.

Was the wrong structure type which caused writing after the allocated
size and was generally bad and wrong.

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>

diff --git a/src/bin/e_desk.c b/src/bin/e_desk.c
index 8c97364..854946a 100644
--- a/src/bin/e_desk.c
+++ b/src/bin/e_desk.c
@@ -224,7 +224,7 @@ e_desk_show(E_Desk *desk)
    E_Event_Desk_Show *ev;
    E_Event_Desk_Before_Show *eev;
    E_Event_Desk_After_Show *eeev;
-   Edje_Message_Float_Set *msg;
+   Edje_Message_Int_Set *msg;
    Eina_List *l;
    E_Shelf *es;
    int was_zone = 0, x, y, dx = 0, dy = 0;
-- 
cgit v0.10.1

